src/App.php

```diff
@@ -606,6 +606,20 @@ public function execute(Route $route, Request $request): self
                 }
             }
         } catch (\Throwable $e) {
+            if (self::$errors === ['*' => []]) { // If no error handler are set
```

Comment:
Not 100% sure about this check, can we just check for `!empty(self::$errors['*'])`? We could also avoid the big `if` condition by just overwriting the empty array with our own callback and leave the default behavior as is.